gen-apidocs/generators/html.go

```diff
@@ -571,29 +565,35 @@ func (h *HTMLWriter) generateHTML(navContent string) {
 
 	for _, sec := range h.TOC.Sections {
 		fmt.Printf("Collecting %s ... ", sec.File)
-		content, err := ioutil.ReadFile(filepath.Join(*api.ConfigDir, "includes", sec.File))
+		content, err := ioutil.ReadFile(filepath.Join(api.IncludesDir, sec.File))
 		if err == nil {
 			buf += string(content)
+			fmt.Printf("\033[32mOK\033[0m\n")
```

Comment:
nit: Create static strings to represent: `Not Found` and `OK`. Replace literals.

Reply:
Done